src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java

```diff
@@ -479,7 +453,7 @@ private void performUploads(Map<String, String> metadata, String bucketName,
           annotateObject(object, listener);
 
           // Log that we are uploading the file and begin executing the upload.
-          listener.getLogger().println(module.prefix(
+          listener.getLogger().println(StorageUtil.prefix(moduleName(),
```

Comment:
Actually, now that all usages of the "prefix" method are in places where "module" is accessible, we could just use module.prefix(), or do you intend to make use of StorageUtil.prefix elsewhere in follow-up changes?

Reply:
I originally wanted to differentiate between Upload and Download in the prefix. On second thought, it probably doesn't make sense. I've updated the Messages.properties instead.
I've amended the last change.